boards/arm/mec172xevb_assy6906/mec172xevb_assy6906.dts

```diff
@@ -222,6 +223,16 @@
 	pinctrl-names = "default";
 };
 
+&eeprom {
+	status = "okay";
+
+	pinctrl-0 = <&eeprom_cs_gpio116
+		      &eeprom_clk_gpio117
+		      &eeprom_mosi_gpio074
+		      &eeprom_miso_gpio075>;
```

Comment:
Question in general: is this EEPROM also accessible from external SPI access? Can it be read / written using dediprog?

Reply:
No